### PR TITLE
fix mew-overlay ui issues

### PR DESCRIPTION
**Pull request by hsurf22**
- remove border around overlay
- remove background color for mobile
- remove box-shadow for mobile
- fix mobile title
- fix small issues
Screenshot from 2021-12-21 18-28-03
Screenshot from 2021-11-02 20-13-58
Screenshot from 2021-11-02 20-14-01
Screenshot from 2021-11-02 20-14-10
Screenshot from 2021-11-02 20-14-13

https://user-images.githubusercontent.com/9893774/140005992-455ac7fe-dc7b-427a-adef-897cbba42f40.mp4

### CODE REVIEW

File: src/components/MewOverlay/MewOverlay.vue [hunk 1]
```diff
@@ -59,35 +65,59 @@
...
+          White sheet (displays on the overlay - size is based on the contentSize prop)
+        =====================================================================================
+        -->
         <v-row class="ma-0 pa-0 d-flex align-center justify-center flex-column">
           <v-sheet
             :width="isMobile ? '100%' : sheetWidth"
             height="100%"
             color="white"
-            :class="['white-sheet-container', isMobile ? 'mt-0' : 'mt-4']"
+            class="pt-5"
```
Comment (olgakup) [inline src/components/MewOverlay/MewOverlay.vue:77]: can you add to the class below please so it looks: 
`            :class="[
              !isMobile ? 'white-sheet-container' : '',
              isMobile ? 'my-0' : 'my-4',
              'pt-5'
            ]"`

File: src/components/MewOverlay/MewOverlay.vue [hunk 2]
```diff
@@ -246,17 +280,27 @@ export default {
...
+
   .mobile-btn {
     z-index: 99999;
   }
 }
-</style>
\ No newline at end of file
+</style>
+
+<style lang="scss">
```
Comment (olgakup) [inline src/components/MewOverlay/MewOverlay.vue:302]: please add this in the same style script, don't create another one

**Review by olgakup — CHANGES_REQUESTED** [review]

File: src/components/MewOverlay/MewOverlay.vue (hunk 2 above)
Comment (hsurf22) [reply src/components/MewOverlay/MewOverlay.vue:302]: This is a global style.

File: src/components/MewOverlay/MewOverlay.vue (hunk 1 above)
Comment (hsurf22) [reply src/components/MewOverlay/MewOverlay.vue:77]: I fixed it by adding  pt-md-0.

**Comment by iluvboba** [comment]
closed since this issue is caused by using v-row on the children components.

**Pull request closed without merging**